nasa/ow_simulator
# Oceanwater 955 delivery command unification

## Linked Issues:
| EPIC ⚡ | [OCEANWATER-933](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-933) |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-955](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-955) |


### Summary of Changes
* Remove goal, feedback, and result components of Deliver action.
* Rename action to TaskDeliverSample.
* Move action file to owl_msgs package.
* Update references to be consistent with new name.

### Test
* This branch must be tested with the corresponding branch in `ow_autonomy`. 
* Do a clean build.  This itself verifies most of renaming and moving.
* Start any simulator world
* Run `task_deliver_sample_action_client.py` and see that the arm performs the deliver movement normally.
